### PR TITLE
hyperrogue: update to 12.1a

### PR DESCRIPTION
**Pull request by tranzystorek-io**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

**Comment by xelxebar** [comment]
@tranzystorek-io Please do! I've been off Void for a few years now.

File: srcpkgs/hyperrogue/template [hunk 1]
```diff
@@ -1,23 +1,38 @@
...
-hostmakedepends="autoconf automake"
+build_style=gnu-makefile
 makedepends="glew-devel libpng-devel SDL-devel SDL_gfx-devel SDL_mixer-devel
  SDL_ttf-devel zlib-devel"
+depends="dejavu-fonts-ttf"
 short_desc="SDL rogue-like in a non-euclidean world"
-maintainer="B. Wilson <x@wilsonb.com>"
+maintainer="Marcin Puc <tranzystorek.io@protonmail.com>"
 license="GPL-2.0-or-later, CC-BY-SA-4.0, CC-BY-SA-3.0, CC-BY-3.0, Public Domain"
 homepage="https://www.roguetemple.com/z/hyper/"
```
Comment (classabbyamp) [inline srcpkgs/hyperrogue/template:12]:
```suggestion
homepage="https://www.roguetemple.com/z/hyper/"
changelog="https://raw.githubusercontent.com/zenorogue/hyperrogue/master/changelog.txt"
```

File: srcpkgs/hyperrogue/files/hyperrogue.desktop [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
```
Comment (classabbyamp) [inline srcpkgs/hyperrogue/files/hyperrogue.desktop:1]: can you try to upstream this?

File: srcpkgs/hyperrogue/template [hunk 3]
```diff
@@ -1,23 +1,38 @@
 # Template file for 'hyperrogue'
 pkgname=hyperrogue
-version=11.3f
+version=12.0u
```
Comment (classabbyamp) [inline srcpkgs/hyperrogue/template:3]: 12.0z is available

File: srcpkgs/hyperrogue/files/hyperrogue.desktop (hunk 2 above)
Comment (tranzystorek-io) [reply srcpkgs/hyperrogue/files/hyperrogue.desktop:1]: I can, but upstream has been generally unresponsive to PRs

File: srcpkgs/hyperrogue/template (hunk 3 above)
Comment (tranzystorek-io) [reply srcpkgs/hyperrogue/template:3]: I saw the 'z' on itch, but don't really see it in the github tags or releases 🤔

Comment (classabbyamp) [reply srcpkgs/hyperrogue/template:3]: well then 🤷‍♀️

File: srcpkgs/hyperrogue/files/hyperrogue.desktop (hunk 2 above)
Comment (tranzystorek-io) [reply srcpkgs/hyperrogue/files/hyperrogue.desktop]: Nevermind, just landed 😄

**Pull request merged**